website/static/js/filepage/revisions.js

```diff
@@ -86,12 +90,31 @@ var FileRevisionsTable = {
             });
         };
 
+        self.showModal = function(hashName, index, hash) {
+            var showModal = bootbox.dialog({
+                title: 'Version ' + index + ' ' + hashName + ' hash',
+                message: '<div class="row">  ' +
+                    '<div class="col-md-12">' +
+                    '<div class="control-group">' +'<p class="col-xs-10">' + hash + '</p>' +
+                    '<a class="btn-sm btn-primary col-xs-2" style="align: right; text-align: center; ' +
+                    'padding: 5px; width: 80px" id="copyBtn" data-clipboard-text="' +
```

Comment:
The hardcoded width was the best way I found to make the 'copy' button behave as expected, but I can look into better solutions. I'm assuming you are talking about the hashes button here though, not the copy button.